packages/cli/src/commands/new/csharp.ts

```diff
@@ -33,10 +33,7 @@ export function CSharp(_: IContext): CommandModule<{}, z.infer<typeof ArgsSchema
           describe: 'the apps name',
           demandOption: true,
           coerce: (name: string) => {
-            return changeCase.pascalCase(
-              name.trim(),
-              { delimiter: '.' }
-            );
+            return changeCase.pascalCase(name.trim());
```

Comment:
dot no longer needed?

Reply:
the delimiter introduces this weird behaviour where a name like "BotBuilder" is converted to "Bot.Builder" which is not what we want here.